### PR TITLE
Table selection menu

### PR DESCRIPTION
**Pull request by cbarbozaerasmus**

### CODE REVIEW

**Comment by ginberg** [comment]
hi @cbarbozaerasmus, is this PR fixing a github issue?

File: R/reportGenerator.R [hunk 1]
```diff
@@ -41,34 +48,37 @@ reportGenerator <- function() {
...
         )
+      )
       ),
 
-    fluidRow(
-      # Item selection menu
-      uiOutput("itemSelectionMenu")
-
-    ),
+    # fluidRow(
```
Comment (ginberg) [inline R/reportGenerator.R:54]: please remove all the code that you have commented out. It's harder to read the code now and if you ever want to enable this code again, you can see it in the github history

Comment (cbarbozaerasmus) [reply R/reportGenerator.R]: Done!

**Comment by cbarbozaerasmus** [comment]
> hi @cbarbozaerasmus, is this PR fixing a github issue?

Deleted the commented code and added this PR to a new issue. Hopefully is ready. Thanks!

**Review by ginberg — APPROVED** [review]

**Pull request merged**